shared/src/common/types/permission/permission-get-all-response-dto.type.ts

```diff
@@ -0,0 +1,7 @@
+import { PermissionsGetAllItemResponseDto } from '~/common/types/permission/permission';
+
+type PermissionsGetAllResponseDto = {
+  items: PermissionsGetAllItemResponseDto[];
+};
```

Comment:
This case is same to mine, where I had to get UsersGetAllResponseDto, there was folowing type UsersGetAllResponseDto = UsersGetAllItemResponseDto[]. I think here should be done same. @Liza-Veis am I right?